data_import/models.py

```diff
@@ -159,7 +129,7 @@ def generate_key(self, request):
 
     @property
     def is_public(self):
-        return is_public(self.user.member, self.source)
```

Comment:
I updated DataFile & ProjectDataFile because of this. The updated models now use a database query rather than the circuitous "look up from a string" approach. That approach is still in the code because a couple other places still use it, but hopefully on the way to full removal. ;)